proofs/signatures/th_lra_test.plf

```diff
@@ -1,4 +1,5 @@
 ; Depends On: th_lra.plf
+;; Polynomial proof
```

Comment:
Can you provide a short explanation before each test saying what does it prove? In this example, it would be something like:
```suggestion
;;This is a proof that the following set of constraints in UNSAT in LRA:
; 2 + (-x) + (-1/2)y >=0
; -8 + x + y >=0
; 0 + x - y >=0
```

Reply:
That's a good idea. I'll do so!